worker/src/RTC/SimulcastConsumer.cpp

```diff
@@ -813,7 +808,12 @@ namespace RTC
 			this->tsOffset = tsOffset;
 
 			// Sync our RTP stream's sequence number.
-			this->rtpSeqManager.Sync(packet->GetSequenceNumber() - 1);
+			// If previous frame has not been sent completely when we switch layer,
+			// We can tell libwebrtc that previous frame is uncompletely by skipping one RTP sequence num.
+			// 'packet->GetSequenceNumber() -2' may increase SeqManager::base and increase the output
+			// seqence num. 
+			// https://github.com/versatica/mediasoup/issues/408
+			this->rtpSeqManager.Sync(packet->GetSequenceNumber() - (this->lastSentPacketHasMarker ? 1 : 2));
```

Comment:
So this is gonna produce a NACK right?

Reply:
can you confirm it @penguinol?

That is the price to pay for this approach.

Reply:
Yes, but it's better than a broken video